projects/igniteui-angular/src/lib/core/styles/themes/generators/_base.scss

```diff
@@ -95,6 +95,10 @@
         --ig-spacing-block-large: 1;
     }
 
+    #{$scope} {
+        --igx-vhelper-scrollbar-size: 1rem;
+    }
+
```

Comment:
Register this as a custom property using the `@property` syntax as done for `--_progress-integer`. You can do this in the core.scss file instead of here.